client/src/components/Navbar/Navbar.tsx

```diff
@@ -90,6 +94,24 @@ const Navbar: React.FC = () => {
               </MenuItem>
             </Menu>
           </>
+        ) : (
+          <>
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Link to={'/signup'} className={classes.linkButton}>
+                BECOME A SITTER
+              </Link>
+
+              <Button variant="outlined" size="medium" onClick={() => history.push('/login')}>
```

Comment:
Try to avoid manipulating the history stack unless it's necessary. You can use react-router Link's for routing here 

Eg: `<Button component={Link} to='/login'....>`